lib/hardware_interface/serial_groundstation_interface.dart

```diff
@@ -117,12 +117,15 @@ class SerialGroundstationInterface extends BaseHardwareInterface {
     var messageDict = parseMessage(packetType, packet);
     packetDict.addAll(messageDict);
 
+    print("messageDict: $messageDict");
```

Comment:
Do we want to keep these prints here? (ok if yes, just checking if they were accidentally left in)